ztp/ran-crd/site-config-crd.yaml

```diff
@@ -244,8 +244,8 @@ spec:
                       ignitionConfigOverride:
                         description: Json formatted string containing the user overrides for the initial ignition config.
                         type: string
-                      proxy:
-                        description: Proxy defines the proxy settings for the cluster. If unset, the cluster will not be configured to use a proxy.
+                      proxySettings:
```

Comment:
I wonder if it is a good idea to break the API since some customer may use the `proxy` field and now with the change to `proxySettings` it will be a breaking change for them.

I think it is a good idea to change our cluster template and the variable from `proxySettings` to `proxy`. For the [unified siteconfig in the MCE side](https://github.com/sakhoury/siteconfig/blob/main/config/crd/bases/metaclusterinstall.openshift.io_siteconfigs.yaml#L406), the field is also set to `proxy`.